Library/Formula/pidcat.rb

```diff
@@ -1,6 +1,7 @@
 require "formula"
 
 class Pidcat < Formula
+  depends_on "bash-completion" => :recommended
```

Comment:
Is this necessary? Most bash completions can just be installed without any added dependencies. If it actually needs it, the dependency should go above `def install`, not at the top of the formula here.